clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/description/TitusDeployDescription.groovy

```diff
@@ -44,6 +44,7 @@ class TitusDeployDescription extends AbstractTitusCredentialsDescription impleme
   int retries
   int runtimeLimitSecs
   Boolean useApplicationDefaultSecurityGroup = true
+  String interestingHealthProviderNames
```

Comment:
It's a comma-separated String and is processed like shown in here:  https://github.com/spinnaker/orca/blob/master/orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/TitusInterestingHealthProviderNamesSupplierSpec.groovy#L77